crates/node/src/shell/mod.rs

```diff
@@ -1089,8 +1092,11 @@ where
                 }
 
                 // Max block gas
+                let gas_scale = namada::parameters::get_gas_scale(&self.state)
```

Comment:
I think we can avoid reading the same value once again, let's just copy the value from here above

Reply:
done in 037f62f73b6c7fd1e9a73b8d485a67caa8ec0767